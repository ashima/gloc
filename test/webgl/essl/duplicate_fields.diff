--- duplicate_fields.true	2011-10-30 19:16:39.000000000 -0700
+++ duplicate_fields.out	2011-11-17 16:38:56.000000000 -0800
@@ -1 +1,4 @@
-error
+precision highp float;
+struct { int q; };
+struct { float q; };
+void main() { }
