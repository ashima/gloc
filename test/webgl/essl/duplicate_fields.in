precision highp float;
struct { int q; };
struct { float q; };
void main() { }
