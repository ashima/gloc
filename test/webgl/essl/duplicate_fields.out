#line 1 0/* <stdin>#u=0 */
precision highp float;
struct { int q; };
struct { float q; };
void main() { }

